### ISSUE TITLE
Bootstrap-material-design-RTL!

### ISSUE THREAD

**Issue by sadeghbarati** [mdbootstrap/mdb-ui-kit#1161]
hi fezvrasta im iranian guy and i have problem with bootstrap-material-design:
your project dont support rtl and it annoying at all,
iadd main file [bootstrap](http://getbootstrap.com/)
i add [bootstrap-rtl](https://github.com/morteza/bootstrap-rtl),
and add your project to my html file [bootstrap-material-design](https://github.com/FezVrasta/bootstrap-material-design),
every thing or few thing right (cuz i dont check all of your site) but ihave problem with checkboxes radiobutton ,inputs 

thank you pls 
Answer

**Comment by FezVrasta** [comment]
Hi, sorry but I have no idea of how to support RTL. Feel free to send a PR if you want to see it fixed

**Comment by sadeghbarati** [comment]
i found [rtl material](https://github.com/ratnic/bootstrap-material-design-rtl) but same problem 😒😢

**Comment by pi0** [comment]
@riiccardo awesome bootstrap-rtl by morteza is not yet ported to v4. Also you can use [Bootstrap4-RTL](https://github.com/mmdsharifi/Bootstrap4-RTL/blob/master/package.json) BTW There is no magic behind them. Simply add `rtl-css` to your build workflow and it will do things magically. (Also don't forget `dir=rtl` on body tag) This way you can easily make any bootstrap based them or framework RTL compatible ...

**Comment by ghaseminya** [comment]
Hi @riiccardo @pi0 
Are you complete a RTL version of this repo? Are you PR your new version?

**Comment by pi0** [comment]
@ghaseminya  As I said making a RTL version is not a hard thing we just need a postcss plugin. We can start another project making pre-built RTL versions or even adding a special RTL version to this repo build workflow. This is Repo maintainer's decision :))

**Comment by FezVrasta** [comment]
I'm happy to accept PRs against the v4 branch

**Comment by ghaseminya** [comment]
@pi0 yes you are true, it is not hard, but if we can add this availability as new feature to current project so any body that want to use this project with RTL support, feel free about use RTL or not use it!
thanks for @FezVrasta  for accept new PR!

**Comment by mhmdadlan** [comment]
I had the same problem with a template built on this framework, so when u switch to rtl for some odd reason in my case : checkboxes buttons and radio inputs were off alignment.
after researching and debugging for a whole day I found that this is caused by the css property` will-change `on buttons and inputs , which kinda cashes the animations to enable the browser to render it smoothly.
any way I removed this line, and it working for now. but the animation looks crappy. I hope u find a solution for it @FezVrasta and if I did I will make my first pull request eva xD ;
